components/kyma-metrics-collector/pkg/process/public_cloud_info.go

```diff
@@ -44,30 +44,36 @@ func (p Providers) GetFeature(cloudProvider, vmType string) (f *Feature) {
 // LoadPublicCloudSpecs loads string data to Providers object from an env var
 func LoadPublicCloudSpecs(cfg *env.Config) (*Providers, error) {
 	if cfg.PublicCloudSpecs == "" {
+		providerErrorCount.WithLabelValues("spec_missing").Inc()
```

Comment:
`LoadPublicCloudSpecs` happens during the bootup only. If this fails, there is `log.Fatalf`. Hence this is no need to have metrics for all these errors, IMO.

Reply:
I see! Removed